src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -602,7 +606,6 @@ private void updateComponents() {
             setDescription(GroupDescriptions.getDescriptionForPreview());
             setNameFontItalic(false);
         }
-        getDialogPane().lookupButton(ButtonType.OK).setDisable(!okEnabled);
```

Comment:
If I understand the changes correctly, then the ok button is now always enabled as soon as there is some text in the `name` text field. However, we also don't want to allow the user to save her changes if some other essential information is missing (e.g. if the type is "search" we need to have a query).

Reply:
Oh right. I missed that part, will add the logic to check that as well.